wordpress-mobile/WordPress-Android
# Site creation rework: "Creating" screen

Fixes #6929 

Touches up the site-creation "Creating" screen by updating the illustration, highlighting which stage the operation is at and altering the illustration and text when error occurs.

To test:
1. With the app logged in, head over to MySite and tap on the "Switch Site" button
2. Tap on the "+" action on the toolbar
3. Select "Creation WordPress.com site" option from the popup
4. Tap on any of the category cards
5. Tap on any of the themes
6. Input a site title in the first input box and hit "Next"
7. Select a domain from the list
8. Notice the "creating" screen. As the operation progresses the labels should get highlighted in sequence. ~A few seconds after the last step is highlighted, a "Success!" toast should come up.~ Also, while in progress, the hardware Back button will produce a Toast ("wait until completed") and the toolbar back button will be hidden.
